Add note informing users not to install Flutter and Dart separately (#5939)
dart-lang/site-www: src/content/get-dart/index.md

To learn more about the Dart SDK, consult the [Dart SDK overview](/tools/sdk).

:::tip

> If you've installed or plan to [install the Flutter SDK][install-flutter], it
> includes the full Dart SDK and you do not need to install Dart separately.

Reviewer: **suggestion:** 

```suggestion
If you've installed or plan to [install the Flutter SDK][install-flutter], it
includes the full Dart SDK. You don't need to install Dart separately and can skip this guide.
```
